diksha/diksha.controller.js

```diff
@@ -411,7 +390,7 @@ let generateResponseStructure = (rSt, rsps) => {
 			...sec,
 			display: strDisplay,
 			name: sec.display.name.en,
-			searchQuery
+			searchQuery : searchQuery
```

Comment:
`: searchQuery` is redundant

Reply:
this is view all fix, undoing that fix here